SmartTutor/ContentModel/LearningObjects/ChallengeModel/MetricHints/MetricHint.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SmartTutor.ContentModel.LearningObjects.ChallengeModel.MetricHints
+{
+    public class MetricHint
```

Comment:
We don't need MetricHints as a separate class. The ChallengeHints are enough as they present a higher level of abstraction and hold the same data (and even more).

Reply:
This was accidentally left during last rebase.  This will be removed.